README.md

```diff
@@ -109,6 +110,14 @@ Or as an ES6 module, from an .mjs file:
 import * as openpgp from 'openpgp';
 ```
 
+#### Deno (experimental)
+
+Import as an ES6 module, from an .mjs file on a CDN of your choice.
+
+```js
+import * as openpgp from "https://cdn.jsdelivr.net/npm/openpgp@5/dist/openpgp.mjs";
```

Comment:
Maybe deno.land/x can be seen as a more trusted source? This will require a maintaner to register the module at ```https://deno.land/x```, and add a webhook to github.

For now, maybe we can keep it anonymous, with an example showing a local import?

Reply:
Alrighty, yeah. We can do that after merging this.

Reply:
Sweet 👍